src/EnergyPlus/IceThermalStorage.cc

```diff
@@ -1684,11 +1680,10 @@ namespace IceThermalStorage {
         //----------------------------
         int loopNum = this->plantLoc.loopNum;
 
-        Real64 CpFluid = FluidProperties::GetDensityGlycol(state,
-                                                           state.dataPlnt->PlantLoop(loopNum).FluidName,
-                                                           state.dataLoopNodes->Node(this->PltInletNodeNum).Temp,
-                                                           state.dataPlnt->PlantLoop(loopNum).FluidIndex,
-                                                           RoutineName);
+        // BUG? I think this is supposed to be getSpecificHeat, not getDensity
+        Real64 CpFluid = state.dataPlnt->PlantLoop(loopNum).glycol->getDensity(state, 
+                                                                               state.dataLoopNodes->Node(this->PltInletNodeNum).Temp,
+                                                                               RoutineName);
```

Comment:
Yes, this should be specific heat. And this could be moved down to line 1718

            Real64 DeltaTemp = Qice / CpFluid / this->ITSMassFlowRate;

Reply:
Should we do that here? Or separately as a bug fix? Oh, another thing to normalize ... no unparenthesized divisions!

Reply:
Issue #10890 posted

Reply:
I would do this separate as a specific bug fix. You could do it here if you like, as long as the diffs are explained (which should be straight-forward).

Reply:
The thing with doing this here is if anyone ever goes back to see what the change was it would be hard to find it.

Reply:
Yep, I'm in full support of doing this in it's own minimalistic PR rather than adding it in here.